base/appointments/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: training
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: training
+  template:
+    metadata:
+      labels:
+        app: training
+    spec:
+      containers:
+      - name: training
+        image: celeo/zdv-easy-appointments:latest
```

Comment:
Word of advice, get away from using latest tags

Reply:
Some reasons for it: In Kubernetes world, these are meant to be versioned so that if you rollout a bad one, you can easily rollback to the previous tag. If it's all latest, you can't rollback and the pod will remain down until you fix the image. Second is ArgoCD looks for differences between the desired state (this repo) and the live state to reconcile. If you keep latest tags, ArgoCD will never restart the deployment, so the updates will never be applied unless you do it manually. The goal with ArgoCD is to prevent as much manual work in the cluster as possible... secrets are the only exception.

Reply:
Thanks for the info! Fixed.